subsys/mgmt/mcumgr/Kconfig

```diff
@@ -53,6 +287,7 @@ config MCUMGR_SMP_UART
 	  commands to be received over UART without requiring an additional
 	  thread.
 
+if MCUMGR_SMP_UART
```

Comment:
This is redundant with `depends on MCUMGR_SMP_UART` in `MCUMGR_SMP_UART_MTU`.

Reply:
Done, thanks!